### PR TITLE
chore: merge local remote 3

### PR DESCRIPTION
**Pull request by xiaolang124**
1.Update ```appStudio``` plugin to support merging local and remote configs.
2.Reading manifest file remains the same, needs to be updated later when all work has been done.
3.Update view logic to support env parameters, needs to be updated later when all work has been done.

### CODE REVIEW

File: packages/fx-core/src/plugins/solution/fx-solution/v2/provisionLocal.ts [hunk 1]
```diff
@@ -178,6 +174,7 @@ export async function provisionLocalResource(
   }
 
   if (isConfigUnifyEnabled()) {
+    setPostDataForLocal(envInfo!, localSettings);
```
Comment (swatDong) [inline packages/fx-core/src/plugins/solution/fx-solution/v2/provisionLocal.ts:177]: Is `envInfo` modified in `configLocalEnvironment`? If so, move this line after calling `configLocalEnvironment`.

Comment (swatDong) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/provisionLocal.ts:177]: Or... will `setPostDataForLocal` be deleted later?

Comment (xiaolang124) [reply packages/fx-core/src/plugins/solution/fx-solution/v2/provisionLocal.ts:177]: When I'm done not storing the ```localSettings``` file, ```setPostDataForLocal``` will be removed.

**Review by swatDong — DISMISSED** [review]

**Review by dooriya — APPROVED** [review]

File: packages/fx-core/src/plugins/resource/appstudio/plugin.ts [hunk 2]
```diff
@@ -1459,7 +1466,7 @@ export class AppStudioPluginImpl {
         outlineIconContent
       );
 
-      if (app.updatedAt && !isLocalDebug) {
+      if (app.updatedAt && !isLocalDebug && isConfigUnifyEnabled()) {
```
Comment (HuihuiWu-Microsoft) [inline packages/fx-core/src/plugins/resource/appstudio/plugin.ts:1469]: Should here be || instead of &&?

Comment (xiaolang124) [reply packages/fx-core/src/plugins/resource/appstudio/plugin.ts]: Thanks, fixed.

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Review by nliu-ms — APPROVED** [review]

**Pull request merged**